src/Renci.SshNet/ConnectionInfo.cs

```diff
@@ -360,11 +360,13 @@ public ConnectionInfo(string host, int port, string username, ProxyTypes proxyTy
 
             Encryptions = new Dictionary<string, CipherInfo>
```

Comment:
@Rob-Hague @drieseng Should we change the order of encriptions? I don't know why they were sorted in this way but I think there was a reason.

Reply:
The order is important. We should have aes128-ctr,aes192-ctr,aes256-ctr first, then leave the rest as they are (I'm not sure we should use most of the rest...)

For reference, these are the values coming from `man ssh_config`:

```bash
 The supported ciphers are:

       3des-cbc
       aes128-cbc
       aes192-cbc
       aes256-cbc
       aes128-ctr
       aes192-ctr
       aes256-ctr
       aes128-gcm@openssh.com
       aes256-gcm@openssh.com
       chacha20-poly1305@openssh.com

 The default is:

       chacha20-poly1305@openssh.com,
       aes128-ctr,aes192-ctr,aes256-ctr,
       aes128-gcm@openssh.com,aes256-gcm@openssh.com
```

Note: a `Dictionary` doesn't guarantee order by contract, but without removing anything it will enumerate in the order the items were added

Reply:
[RFC4344](https://datatracker.ietf.org/doc/html/rfc4344#section-4) recommends AES-CTR modes, and also 3DES-CTR which we don't have yet. It says AES modes are preferred going forward.
The older [RFC4253](https://datatracker.ietf.org/doc/html/rfc4253#section-6.3) required 3DES-CBC and recommended AES128-CBC.

I've reordered the cyphers accordingly. 3DES-CTR can be added on a separate PR if needed.

Reply:
I'll leave that for a separate PR then.